### PR TITLE
#153 Add Chinese Translations

### PR DESCRIPTION
**Pull request by jiangzhe11**
A new PR

### CODE REVIEW

File: README-Chinese.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+## 屏幕截图
+
+
+| <!-- --> | <!-- --> |
+| - | - |
+| [![Empty interface](https://nitratine.net/posts/auto-py-to-exe/empty-interface.png)](https://nitratine.net/posts/auto-py-to-exe/empty-interface.png) | [![Filled out](https://nitratine.net/posts/auto-py-to-exe/filled-out.png)](https://nitratine.net/posts/auto-py-to-exe/filled-out.png) |
+| [![Converting](https://nitratine.net/posts/auto-py-to-exe/converting.png)](https://nitratine.net/posts/auto-py-to-exe/converting.png) | [![Completed](https://nitratine.net/posts/auto-py-to-exe/completed.png)](https://nitratine.net/posts/auto-py-to-exe/completed.png) |
+
+中文翻译：jiangzhe
```
Comment (brentvollebregt) [inline README-Chinese.md:115]: Do you want to link your GitHub profile?

```suggestion
中文翻译：[jiangzhe](https://github.com/jiangzhe11)
```

**Review by brentvollebregt — APPROVED** [review]

**Comment by brentvollebregt** [comment]
This looks excellent! Thank you very very much 👍

Just wondering if you want to link your name in the translated README to link to your GitHub profile? If not that is fine, I can merge as is then.

**Comment by jiangzhe11** [comment]
Certainly.

Quite happy to see the translation finished. Thanks to your help!

**Comment by Dwalinnewyork** [comment]
[README-Chinese.md](https://github.com/brentvollebregt/auto-py-to-exe/files/6919246/README-Chinese.md)

**Pull request merged**